### PR TITLE
AAP-20543 Ensure UI parity for just a couple of sections for EDA controller  guide

### PR DESCRIPTION
**Pull request by jself-sudoku**
This PR covers just a few sections of the EDA controller user guide since multiple sections are being covered under separate jiras that required changes beyond the scope of UI parity. 

Covered in this PR: 

- downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc (Settting up a new decision environment)
- downstream/modules/eda/proc-eda-set-up-rulebook-activation.adoc (Setting up a rulebook activation)
- downstream/modules/eda/con-eda-rulebook-activation-list-view.adoc (Rulebook activation list view)
- downstream/modules/eda/proc-eda-view-activation-output.adoc (Viewing activation output)
- downstream/modules/eda/proc-eda-view-rule-audit-actions.adoc (Viewing rule audit actions)

### CODE REVIEW

File: downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc [hunk 1]
```diff
@@ -13,8 +13,9 @@ For more information, see the xref:eda-set-up-credential[Setting up credentials]
 
 .Procedure
 // ddacosta I'm not sure whether there will be an EDA specific dashboard in the gateway. Step 1 might need to change to something like "Log in to AAP".
-. Navigate to the {EDAcontroller} Dashboard.
-. From the navigation panel, select {MenuADDecisionEnvironments}.
+. Log in to the {PlatformNameShort} Dashboard.
+. Navigate to {MenuADDecisionEnvironments}.
+. Click btn:[Create decision environment]
```
Comment (sayjadha) [inline downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc:18]: Missing a period towards the end.

**Review by sayjadha — COMMENTED** [review]
One minor comment, rest LGTM.

File: downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc:18]: Thank you, Sayee.

**Review by sayjadha — COMMENTED** [review]
LGTM.

**Pull request merged**